README.md

```diff
@@ -28,6 +28,15 @@ pnpm dev
 # http://localhost:3333
 ```
 
+## docker
+
+```bash
+# build
+docker build . -t henryclw/cook:dev
```

Comment:
> I found out that if I don't add @unocss/reset, it cannot start as expected. If this is not needed, would you mind removing it in the main branch then I merge your newest main branch?

Can you provide more error information? I don't think it will fail to start if installed using npm or pnpm normally.
You can test with vercel/netlify and other CI tools to prove my conclusion.

---

Unlike my other projects, this one is strongly data-bound, and it's a public application rather than a library.

My purpose of making it open source is that everyone can maintain its data and functions together, and make it better for everyone to use.

So I don't want it to be redeployed when everyone uses it, its feedback will be easily scattered.
If you really want to build and deploy it yourself, I'm not against it because of the MIT license.

Since its users are basically in China, I have to keep my record number to ensure it is accessible.

---

Thanks a lot for your contribution, but for the reasons above, I'm not going to merge it for now.
If you have other reasons, let me know.

Reply:
Okay, thank you for your reply. For the `@unocss/reset` dependency, I've opened a new issue, #12